bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=odf-multicluster-orchestrator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.8.0+git
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.10.0+git
```

Comment:
we are using v1.9.

Reply:
Do we need to commit bundle.Dockerfile updates?

Reply:
I updated this to v1.9 , not sure if that helps it.